src/Fantomas.Tests/RecordTests.fs

```diff
@@ -1894,3 +1894,27 @@ let parse (checker: FSharpChecker) (parsingOptions: FSharpParsingOptions) { File
         })
     |> Async.Parallel
 """
+
+[<Test>]
+let ``anonymous records with comments on members, 2067`` () =
+    formatSourceString
+        false
+        """
+{|
+    // The foo value.
+    FooValue = fooValue
+    // The bar value.
+    BarValue = barValue
+|}
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+{|
+// The foo value.
```

Comment:
I'm having an unexpected output regarding the styling, I would guess that I have to change something around [here](https://github.com/fsprojects/fantomas/blob/master/src/Fantomas/CodePrinter.fs#L3053), can you confirm if that's the case? Also, what would be the expected styling/output in this case?

Reply:
Yes, I was expecting it more to be something like
```fsharp
{| // The foo value.
   FooValue = fooValue
   // The bar value.
   BarValue = barValue |}
```
I'll need to look into why this is happening, I'll get back to you.